### PR TITLE
Possible fix for #1398

### PR DESCRIPTION
**Pull request by karanjthakkar**

### CODE REVIEW

**Comment by Garbee** [comment]
Since `handleScroll` is just the same thing as the `mouseLeave` handler, why not make a generic `hideTooltip` function, bind that then call that bound method for both events? That will reduce duplication and keep maintainability easy.

**Comment by Garbee** [comment]
Oh, and this should go into `mdl-1.1` since it is a bug-fix. Otherwise it won't make it into any releases until 2.x.

**Comment by karanjthakkar** [comment]
@Garbee I've refactored the code to make it DRY. Once you find it acceptable, I can send another PR to `mdl-1.1` and close this.

**Comment by karanjthakkar** [comment]
@Garbee Create #4372 for this.

File: src/tooltip/tooltip.js [hunk 1]
```diff
@@ -133,13 +133,15 @@
         }
 
         this.boundMouseEnterHandler = this.handleMouseEnter_.bind(this);
-        this.boundMouseLeaveHandler = this.handleMouseLeave_.bind(this);
+        this.boundMouseLeaveHandler = this.hideTooltip_.bind(this);
+        this.boundScrollHandler = this.hideTooltip_.bind(this);
```
Comment (Garbee) [inline src/tooltip/tooltip.js]: You don't need to bind the same thing twice. Simply do *one* bound handler and re-use it for each event listener. Same result, less memory.

**Pull request closed without merging**